### PR TITLE
Project Mobile first setup

### PR DESCRIPTION
**Pull request by lukademinnosa**
**Hello**

I have finished the mobile first setup and submitted as expected. Please review my work.

Regards.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,180 @@
...
+
+- [Live Demo Link](https://drive.google.com/file/d/1DoA8aY7Xz032Xp8uOKEfkHvtTk0n5mel/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (Ibizugbe) [inline README.md]: Kindly take away these template statements from your readme as they give your project a less professional look😉.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          <a href="#"><i class="fa-brands fa-medium"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+        </li>
+      </ul>
+    </section>
```
Comment (Ibizugbe) [inline index.html:51]: Your background image for the headline section is repeating itself, kindly do well to fix it as it makes your project looks a bit different from the given design😉.

image


Let's also do well to implement the curve at the buttom left corner of the headline section to stick to the deisgn😉
image


Kindly note that it is a requirement in this milestone that our project matches the given design😉.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    </header>
+    <section>
+      <h1>I&rsquo;m Francis.<br />Glad to see you!</h1>
+      <p>
+        I&rsquo;m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded, don&rsquo;t
+        hesitate to contact me.
+      </p>
+      <a href="#" class="connect">LET’S CONNECT</a>
```
Comment (Ibizugbe) [inline index.html:33]: Kindly ensure to make use of the right font family and font weight for your texts so as to make them match the given design 😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

File: index.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  <title>Document</title>
+  <link rel="stylesheet" href="css/style.css" />
+  <script src="https://kit.fontawesome.com/5784b9ea6f.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <nav class="flex-nav">
+        <h1 class="logo"><a href="#">My Logo</a></h1>
```
Comment (Ibizugbe) [inline index.html]: I suggest you take away the `h1` tag wrapping your logo as this makes your page more semantic😉.

```suggestion
        <a href="#" class="logo">>My Logo</a>
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        <li>
+          <a href="#"><i class="fa-brands fa-medium"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+        </li>
+      </ul>
```
Comment (Ibizugbe) [inline index.html:50]: Nice job using flexbox to layout your social links👍. Let's try to make sure it sticks to the design by adjusting the space (margin and padding) around them😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @lukademinnosa 👋👋,

tryagain
 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are passing👍.
- [x] Github flow have been used👍.
- [x] Contents are aligned using flexbox👍.
- [x] Toolbar section is wrapped in a header tag and headline section is wrapped in a section tag👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lukademinnosa** [comment]
Hello @Ibizugbe I have sorted out the issues you highlighted. Hope all is fulfilled. Regards.

**Comment by Ibizugbe** [comment]
Hello @lukademinnosa, Nice hearing that👍. If you are sure everything have been fixed, you can go ahead to request for a review on your dashboard😉.

**Comment by lukademinnosa** [comment]
I did @Ibizugbe . Awaiting a positive review now. Thanks

File: README.md [hunk 6]
```diff
@@ -1,2 +1,177 @@
...
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+A video to the working of the website is loaded below:
+
+- [Live Demo Link](https://drive.google.com/file/d/1M7w158Yel44sGCPEYjtImRorxx8WnuaO/view?usp=drive_link)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (wayungi) [inline README.md:79]: - [OPTIONAL] If your project is not hosted and the requirement does not state that it should be hosted, please delete the live demo section or kindly host and include a link to the live version of the project. Your live demo link is currently linked to a project presentation. Please have this fixed. 🤝

**Review by wayungi — APPROVED** [review]
Hi @lukademinnosa , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] All linter checks are passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] All requested changed implemented ✔️ 

## Optional suggestions

 - [OPTIONAL] Please give your future branches meaningful names. **newBranch** says nothing about the changes you have introduced in this branch. I suggest a descriptive name like **feat-create-header-headline** 🥲 
 
branch

- [ OPTIONAL]  Your PR description should include a summary of what you have done in this PR. You can always know what you have done by checking the project requirements and rewarding them into what you have done. Please note that you should not list the project requirements instead. 🦺 
pr



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**